YakimS/assignment1/Q2

```diff
@@ -0,0 +1,89 @@
+public class Assignment1Q2 {
+
+    public static void main(String[] args) {
+        MyLinkedList<Integer> myLinkedList = new MyLinkedList<>();
+        Integer[] ob = new Integer[10];
+        for(int i = 0 ; i < ob.length ; i++)
+        {
+            ob[i] = i;
+            myLinkedList.add(ob[i]);
+            System.out.print(ob[i] + "");
+        }
+        System.out.println("");
+        System.out.print("kth ele:"+myLinkedList.getKthLastElement(9));
+    }
+
+
+    public static class MyLinkedList<T>
```

Comment:
This class as well as `MyNode` has really nice API. Good job!

Could you add some comments to explain what methods do?

Reply:
No problem